crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1950,6 +1956,8 @@ impl<'db> KnownInstanceType<'db> {
             Self::Union => KnownClass::SpecialForm,
             Self::NoReturn => KnownClass::SpecialForm,
             Self::Never => KnownClass::SpecialForm,
+            // typing.Any is _not_ currently defined as a _SpecialForm in the typeshed.
```

Comment:
I would be inclined to leave out this comment, since a) it's not the only KnownInstance which isn't defined as `_SpecialForm`, b) the fact that it isn't an instance of `_SpecialForm` is self-evident here, and c) it's unlikely to be changed to be an instance of `_SpecialForm`.
```suggestion
```